ui/src/components/header/ProjectMenu.vue

```diff
@@ -81,7 +81,7 @@ export default {
       const projects = []
       const getNextPage = () => {
         this.loading = true
-        api('listProjects', { listAll: true, page: page, pageSize: 500, showIcon: true }).then(json => {
+        api('listProjects', { listAll: true, page: page, pageSize: 500, details: 'min', showIcon: true }).then(json => {
```

Comment:
@rohityadavcloud, how this line is related to the issue and the PR's proposal?

Reply:
Something I spotted, makes the listing faster. We use details=min in many other places.